src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +145,55 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const $entry = $(e.target);
+        const studentEmail = $entry.parentsUntil('.fslink_student').find('input[name=studentEmail]').val();
+        const subjectType = $entry.find('input[name=subjectType]').val();
+        const courseName = $entry.find('input[name=courseName]').val();
+        const courseId = $entry.find('input[name=courseId]').val();
+        const studentName = $entry.find('input[name=studentName]').val();
+        const sessionStatus = $entry.find('input[name=sessionStatus]').val();
+        const sessionName = $entry.find('#feedback-session-name').text();
+        const sessionNameWithoutDate = sessionName.substring(0, sessionName.lastIndexOf(' ['));
+
+        const relatedLink = encodeURIComponent($entry.find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = `${ParamsNames.TEAMMATES_EMAIL_ADDRESS}`;
+
+        const bodyHeader = `Hello ${studentName},`;
+        const bodyFooter = '%0D%0A%0D%0AIf you encounter any problems when using the system,'
+                         + ` you can email TEAMMATES support team at ${supportEmail}.`
+                         + '%0D%0A%0D%0ARegards,%0D%0ATEAMMATES Team.';
```

Comment:
On the topic of custom indentation, it might be better to start the string on the next line, similar to what is being done for `bodyContent` later in the file.